homepage.html

```diff
@@ -0,0 +1,313 @@
+              <!-- gonna represent share & like dislike ....-->
+              <div class="left-most">
+                <a href="#">
+                  <i class="fas fa-plus"></i><span>Add to </span>
+                </a>
+
+                <a href="#">
+                  <i class="fas fa-share"></i><span>Share</span>
+                </a>
+
+                <a href="#">
+                  <i class="fas fa-ellipsis-h"></i><span>More</span>
+                </a>
+
+              </div>
+
+              <div class="right-mostUP">
+                <a href="#">
+                  <i class="fas fa-thumbs-up"></i><span>359</span>
+                </a>
+
+                <a href="#">
+                  <i class="fas fa-thumbs-down"></i><span>14</span>
+                </a>
+
+              </div>
+
+            </div>
+
+
+          </section>
+        <!--description section-->
+        <section class="description">
+          <h4>Published on Feb 27, 2014</h4>
+          <p>How do you contribute to open source? In this video, I'll describe the easiest way to make a simple contribution to an open source project -- without using the command line, a text editor, "forking a repo", or even touching Git outside of Github. I'll also describe what kinds of contributions open source software projects are typically looking for. As an example, I use submitting a student solution to a web development exercise from The Odin Project, a free online curriculum for learning web development with Ruby on Rails. You can find more information about that project at http://theodinproject.com.
+          </p>
+            <hr class="main-horizontal-line">
+            <div class="main-show-more">
+              <a href="#">
+                SHOW MORE
+              </a>
+            </div>
+        </section>
+
+        <!--comments section-->
+        <section class="comments">
+          <img src="./assets/imgs/loading.gif"  alt="loading">
+          <p>Loading...</p>
+        </section>
+      </section>
+
+      <aside>
+        <div class="control-recommended-video-list">
+          <div class="left-most"><p><strong>Up next</strong></p></div>
+          <div class="right-most">
+            <p>Autoplay </p>
+            <img src="./assets/imgs/info.png" alt="information" >
+            <i class="fas fa-toggle-on"></i>
+          </div>
+        </div>
+        <div class="preview-container">
+          <div class="thumbnail">
+            <a href="#"><img src="./assets/imgs/preview1.png" alt="First recommended Video thumbnail" > </a>
+          </div>
+          <div class="preview-info">
+              <a href="#"><p class="preview-videoTitle"><strong>Contributing to Open Source Part II: The Real Way</strong></p></a>
+              <a href="#"><p class="preview-channel-name">The Odin Project</p></a>
+            <p class="preview-views">29,985 views</p>
+          </div>
+        </div>
+
+
+        <ul>
+          <li><hr id="previewContainer-hLine"/></li>
```

Comment:
## Using an ID Selector for Multiple Elements[1]
When writing CSS, there should only be one ID in the HTML. Always use the class attribute for multiple elements.
- [ ] Use `class` instead `id`

_[1] [Good and Bad CSS practices](https://speckyboy.com/good-bad-css-practices/)_